### PR TITLE
Us 05 bottom navigator

### PR DESCRIPTION
**Pull request by giahy1**
Us 05 bottom navigator

### CODE REVIEW

File: lib/src/screens/home_screen.dart [hunk 1]
```diff
@@ -1,10 +1,62 @@
...
+      bottomNavigationBar: BottomNavigationBar(
+        showSelectedLabels: false,
+        showUnselectedLabels: false,
+        items: <BottomNavigationBarItem>[
+          BottomNavigationBarItem(
+              icon: Image(
+                image: AssetImage('assets/images/search_icon.jpg'),
+                color: _selectedIndex == 0 ? AppColor.green : AppColor.iconText,
+              ),
+              label: 'Search'),
```
Comment (duongvanquang) [inline lib/src/screens/home_screen.dart]: Can I leave the label blank ?

Comment (giahy1) [reply lib/src/screens/home_screen.dart]: no u can't . Firstly, it is used for widget test easily. Secondly, It shows the meaning of the icon

File: lib/src/screens/home_screen.dart [hunk 2]
```diff
@@ -1,10 +1,62 @@
 import 'package:flutter/material.dart';
```
Comment (thanhphamtrung) [inline lib/src/screens/home_screen.dart:1]: should we call this file is "NavigationScreen"?

File: lib/src/screens/home_screen.dart (hunk 1 above)
Comment (duongvanquang) [reply lib/src/screens/home_screen.dart]: ok i agree

File: lib/src/screens/home_screen.dart (hunk 2 above)
Comment (giahy1) [reply lib/src/screens/home_screen.dart:1]: I renamed it to NavigationScreen

Comment (giahy1) [reply lib/src/screens/home_screen.dart:1]: I renamed it to NavigationScreen

**Pull request merged**